tests/test_auditing.py

```diff
@@ -385,3 +392,106 @@ def test_create_commit_message(migration_model: capellambse.MelodyModel):
         "- connector v1\n"
         f"- RM-Bridge v{__version__}\n"
     )
+
+
+@pytest.mark.integtest
+def test_create_commit_message_on_no_changes(
+    migration_model: capellambse.MelodyModel,
+):
```

Comment:
This one's interesting: If there were no changes, there should - to my understanding - be no new commit. Then why would we want to generate a commit message?

Reply:
The commit message is also printed to stdout. So for no changes there will be no commit but the printing of the message.